src/ext.js

```diff
@@ -403,6 +403,17 @@ const getDefinition = (env) => {
                         },
                         defaultValue: env?.anything?.sense?.isUnsupportedFeature?.('totals') ? 'None' : 'Expr',
                       },
+                      totalsAggrFuncMessage: {
+                        component: 'text',
+                        translation: 'Object.Table.AggrFuncMessage',
+                        show(data, handler) {
+                          return (
+                            handler &&
+                            !handler.properties.totals.show &&
+                            handler.properties.totals.position === 'noTotals'
```

Comment:
Is it working. Is it all that should be kept?

Reply:
also if you switch the order you don't need to check handler first
```suggestion
                            handler.properties.totals.position === 'noTotals' &&
                            !handler.properties.totals.show

```

Reply:
Data does not include properties but handler does. 
As far as I tested it always exists, it just might be shown or not regarding the conditions.
`totals.show` is in higher priority here since with `show = true` it doesn't need to go for checking position at all,  position can stay noTotals in both cases.

Reply:
ok then handler works fine.
Well if `position` is not 'noTotals' there is no need to check `show`. My point is that you can remove the first condition on line 411

Reply:
ok, right. I'll switch them.